src/components/Uploader/style.css

```diff
@@ -52,3 +52,8 @@
     margin-top: 4.5rem;
   }
 }
+
+.clientLgScreen {
+  min-height: 16.5rem;
```

Comment:
Why do we need this?

Reply:
This is to make sure that the uploader screen on the cross device client looks good if it's opened on desktop or large screens